### DISCUSSION TITLE
Dịch skip connection

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#148] [Terminology] [answered]
Context:
> The key to being able to train such a deep network is to use **skip connections** (also called shortcut connections): the signal feeding into a layer is also added to the output of a layer located a bit higher up the stack.

Mình đề xuất dịch từ này là "kết nối tắt".

@mlbvn/handson-ml

**Comment by thanhcsf** [comment, marked as answer]
Anh thấy từ này dịch **kết nối tắt** thì hợp lý rồi.
